var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -634,23 +644,28 @@ def edit(self, spec, prefix):
 
             fcflags += ["-I{0}".format(join_path(elpa_incdir, "modules"))]
 
-            # Currently AOCC support only static libraries of ELPA
-            if "%aocc" in spec:
-                libs.append(
-                    join_path(
-                        elpa.prefix.lib, ("libelpa{elpa_suffix}.a".format(elpa_suffix=elpa_suffix))
-                    )
+            # Currently AOCC and FJ support only static libraries of ELPA
+            if "%aocc" or "%fj" in spec:
+                libs += (
```

Comment:
@alalazo 
I switched from .append to += because using .append resulted in an AttributeError, stating that 'LibraryList' object has no attribute 'append.
```
==> [2023-11-29-19:52:04.859849] Error: AttributeError: 'LibraryList' object has no attribute 'append'
```
It seems that the type of 'libs' has changed from a list to 'LibraryList' at some point. If there is a better way to handle this, I would appreciate your advice."